Backend/app/dashboard/Lib/src/Storage/Queries/Person.hs

```diff
@@ -271,3 +272,48 @@ instance ToTType' BeamP.Person Person.Person where
         mobileNumberHash = mobileNumber.hash,
         ..
       }
+
+findAllByFromDateAndToDateAndMobileNumberAndStatusWithLimitOffset ::
+  (BeamFlow m r, EncFlow m r) =>
+  Maybe UTCTime ->
+  Maybe UTCTime ->
+  Maybe Text ->
+  Maybe FleetOwnerStatus ->
+  Maybe Int ->
+  Maybe Int ->
+  m [Person]
+findAllByFromDateAndToDateAndMobileNumberAndStatusWithLimitOffset mbFromDate mbToDate mbMobileNumber mbStatus mbLimit mbOffset = do
+  mbMobileNumberDbHash <- traverse getDbHash mbMobileNumber
+  findAllWithOptionsDb
+    [ Se.And
+        ( [ Se.Or
+              [ Se.Is BeamP.verified $ Se.Eq (Just False),
+                Se.Is BeamP.verified $ Se.Eq Nothing
+              ]
```

Comment:
I think you should remove this condition, because it would work wrong for `mbStatus = Just Approved`

this one is enough `[Se.Is BeamP.verified $ checkStatus | isJust mbStatus]`